pkg/cmd/cli/cmd/project.go

```diff
@@ -66,7 +66,6 @@ func NewCmdProject(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.
 		Short:   "Switch to another project",
 		Long:    projectLong,
 		Example: fmt.Sprintf(projectExample, fullName),
-		Aliases: []string{"projects"},
```

Comment:
> I mean uses by the world at large, which we can't refactor :)

I suspect that @fabianofranz may take it anyway if he thinks its a usability gain.  Realistically the exposure is small, non-scripts may expect this behavior anyway and `oc project` was never something reason to script against.